File: src/modulo3/Aula17/P1/Conta.java
package modulo3.Aula17.P1;

public class Conta{
    private double saldo;
    private String codigoCliente;

    public Conta(double saldoInicial, String cliente){
      this.saldo = saldoInicial;
      this.codigoCliente = cliente;
    }

    public void deposito(double valor){
        this.saldo+= valor;
    }

    public void saque(double valor){
        this.saldo -= valor;
    }
    public double getSaldo(){
        return saldo;
    }
}

File: src/modulo4/aula006JS/Conta.js
export class conta{
    agencia;
    numero;
    _saldo;
    cliente;
 
get Agencia(){
    return this.agencia;
}

set agencia(ag){
    this.agencia = ag;
}

depositar(valor){
    this._saldo +=valor;
}

saque(valor){
    if(this.saldo > valor){
    this._saldo -=valor;
    }
    return "saldo insuficiente";
    }

}